.expeditor/config.yml

```diff
@@ -0,0 +1,64 @@
+product_key: chef-server
+
+# Slack channel in Chef Software slack to send notifications about build failures, etc
+slack:
+  notify_channel: chef-server-notify
+
+github:
+  # The file where the MAJOR.MINOR.PATCH version is kept. The version in this file
+  # is bumped automatically via the `built_in:bump_version` merge_action.
+  version_file: "VERSION"
+  # The file where our CHANGELOG is kept. This file is updated automatically with
+  # details from the Pull Request via the `built_in:update_changelog` merge_action.
+  changelog_file: "CHANGELOG.md"
+  # The Github Team primarily responsible for handling incoming Pull Requests.
+  maintainer_group: chef/chef-server-maintainers
+
+# These actions are taken, in order they are specified, anytime a Pull Request is merged.
+merge_actions:
+  built_in:bump_version:
+    ignore_labels: "Version: Skip Bump"
+  built_in:update_changelog:
+    ignore_labels: "Changelog: Skip Update"
+  built_in:trigger_omnibus_release_build:
+    ignore_labels: "Omnibus: Skip Build"
+    only_if: built_in:bump_version
+
+# These actions are taken, in the order specified, when an Omnibus artifact is promoted
+# within Chef's internal artifact storage system.
+#
+# TODO: add promoted_to_unstable action to update changelog with modified omnibus components
+artifact_actions:
+  promoted_to_unstable:
+    bash:.expeditor/update_acc.sh:
```

Comment:
@srenatus I'll let @tduffield correct me if I'm wrong but these can be a list...for example:
https://github.com/chef/automate/blob/master/.expeditor/config.yml#L9-L13